swagger.yaml

```diff
@@ -3124,6 +3122,21 @@ definitions:
               format: uuid
             hostname:
               type: string
+      disks_selected_config:
+        type: array
```

Comment:
@avishayt the something is not clear to me, we have an array parameter to set selected installation disk but only one installation disk path `installation_disk_path` so what is the point having an array paramter?